lib/control-connection.js

```diff
@@ -95,10 +95,12 @@ ControlConnection.prototype.init = function (callback) {
           var portSeparatorIndex = name.lastIndexOf(']:');
           return addHost(name.substr(1, portSeparatorIndex - 1), name.substr(portSeparatorIndex + 2), eachNext);
         }
-        if (name.indexOf('.') > 0 && name.indexOf(':') > 0) {
+        if (name.indexOf(':') > 0) {
```

Comment:
@tolbertam I see what you mean by the comment on line 99.

However if you trace the code execution you will see that the same condition was the way that `my.domain:9999` was getting "added" by the call to `addHost` inside that `if` condition and it is the reason why `my-domain:9999` is not getting added (if this fix is not applied).

This should not impact the IPv4 functionality of this section of code either. This change will allow all of these patterns to be added via the `addHost` call:
```
my-host:9999
my.host:9999
1.2.3.4:9999
```

Let me know what you think.